src/main/kotlin/app/revanced/util/AbstractIntegrationsPatch.kt

```diff
@@ -0,0 +1,39 @@
+package app.revanced.util
```

Comment:
What is the motivation this is in util?

Reply:
Made this while working on another PR (coming soon:tm:) and I had placed my other classes there as well (those were actual utils), so that's probably why. Do you have a better place/package in mind?

Reply:
This is currently a bit problematic with the packaging, since we package per app, it is not possible to have 1 package for patches of each of those packages. Currently, there is a shared package which hosts a shared fingerprint, other than that I am not sure how else to package.

Reply:
Should I keep it like this for now then, until a solution has been found? I don't see any point in delaying this PR for something small like this.